pages/glossary/default-page-language.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Default Page Language
+key: default-page-language
+unambiguous: true
+objective: true
+input_aspects:
+  - DOM tree
+  - Accessibility tree
+  - CSS Styling
+  - Language
+---
+
+The default page language is the _most common language_ used in a [web page][]. If there are no _words_ in the page, the default language is undefined.
+
+The _most common language_ is determined by counting the number of _words_ in the [web page][] that are part of any of the languages in the [language subtag registry][]. The same word can be part of multiple languages. If the number of words is the same, the default language is undefined.
+
+The _words_ used in the following texts should be counted for the _most common language_:
+
+- **page title**: The value of the [document title][] of the [document][] in the [top-level browsing context][].
+- **text nodes**: The value of any [text nodes][] that is [visible][] or [included in the accessibility tree][]
+- **accessible text**: The [accessible name][] and [accessible description][] of any element included in the [accessibility tree][]
+
+**Exception**: Words in text that comes from nodes that have an [ancestor][] in the [flat tree][] with a non-empty (`""`) `lang` attribute, except for the [root node][].
```

Comment:
Do you mean "non-empty lang" or "valid lang"?
i.e., do you mean to exclude `<p lang="xyz">Lorem ipsum</p>`?
I am not sure how invalid lang are handled by UA/AT. If they default to the parent lang, then these should not be ignored from the count.

Reply:
AT are inconsistent about what they do, some default to the parent, others default to the browser's default language. That's why I think it's best to keep them out. Clearly someone meant for that to be a different language.